File: tradeit-android-sdk/src/main/java/it/trade/android/sdk/model/TradeItCallbackWithSecurityQuestionAndCompletion.java
package it.trade.android.sdk.model;

import it.trade.model.TradeItSecurityQuestion;
import it.trade.model.callback.TradeItCallbackWithSecurityQuestionImpl;


public interface TradeItCallbackWithSecurityQuestionAndCompletion extends TradeItCallBackCompletion{
    void onSecurityQuestion(TradeItSecurityQuestion securityQuestion, TradeItCallbackWithSecurityQuestionImpl callback);
}

File: tradeit-android-sdk/src/main/java/it/trade/android/sdk/model/TradeItCallbackWithSecurityQuestionAndCompletion.kt
package it.trade.android.sdk.model

import it.trade.model.TradeItSecurityQuestion
import it.trade.model.callback.TradeItCallbackWithSecurityQuestionImpl


interface TradeItCallbackWithSecurityQuestionAndCompletion : TradeItCallBackCompletion {
    fun onSecurityQuestion(securityQuestion: TradeItSecurityQuestion, callback: TradeItCallbackWithSecurityQuestionImpl<*>)
}
